cfe_ts_crc.c

```diff
@@ -125,10 +126,16 @@ int main(int argc, char **argv)
     if (fd < 0)
     {
         printf("\ncfe_ts_crc error: can't open input file!\n");
-        exit(0);
+        exit(1);
     }
     /* seek past the number of bytes requested */
-    lseek(fd, skipSize, SEEK_SET);
+    offsetReturn = lseek(fd, skipSize, SEEK_SET);
+    if (offsetReturn != skipSize)
+    {
+        printf("\ncfe_ts_crc error: lseek failed!\n");
+        printf("%s\n", strerror(offsetReturn));
```

Comment:
`strerror()` takes an errno, not the return code (the latter is -1 by definition).

So it should be `strerror(errno)` ... OR just use `perror()` routine instead.  That just simplifies when you want to print the current errno.

We should also do this EVERY time we do `exit(1)` - not just here.

I do recommend using this function when all you want to do is print the error in the simplest way possible.  It's quick and easy.  See: https://man7.org/linux/man-pages/man3/perror.3p.html